schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+            "items": {
+              "$ref": "#/definitions/affiliation"
+            },
+            "minItems": 1
+          },
+          "orcid": {
+            "title": "ORCID iD",
+            "allOf": [
+              {
+                "type": "string",
+                "pattern": "^\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]$"
+              },
+              {
+                "type": "string",
+                "$comment": "TODO: consider adding 'orcid' format here; however, some validators hang on unexpected formats. Would need to implement validator using checksum method described in https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier"
+              }
+            ]
+          },
+          "author_id": {
+            "description": "arXiv author identifier, if available.",
+            "oneOf": [
+              {
+                "type": "string",
+                "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$"
+              },
+              {"type": "null"}
+            ]
+          },
+          "is_submitter": {
+            "description": "Indicates whether this person is the submitter of this submission",
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "full_name",
+          "first_name",
+          "last_name"
+        ]
+      },
+      "arxiv_status": {
+        "title": "arXiv posting status",
+        "description": "The status of this document version on arXiv.",
+        "type": "string",
+        "enum": ["posted", "withdrawn"],
+        "minLength": 1
+      },
+      "external_publication_status": {
+        "title": "External publication status",
+        "description": "The publication status of this document version.",
+        "type": "string",
+        "enum": ["submitted", "accepted", "published"],
+        "$comment": "Is this enough?"
+      },
+      "reference_string": {
+        "title": "Reference string",
+        "description": "An unstructured reference string.",
+        "type": "string",
+        "minLength": 1
+      },
+      "source_info": {
+        "title": "Source information",
+        "description": "Information about the source input.",
+        "type": "object",
+        "properties": {
+          "source_format": {
+            "title": "Source format",
+            "type": "string",
+            "enum": ["tex", "pdftex", "pdf", "ps", "html", "docx", "none"]
+          },
+          "legacy_source_flags": {
+            "title": "Legacy source flags",
+            "type": "string",
+            "enum": ["1", "1A", "1S", "A", "AD", "AS", "D", "S", "S1"]
+          },
+          "size_kilobytes": {
+            "title": "Source size in kilobytes",
+            "description": "The size of the source file, in kilobytes.",
+            "type": "number"
+          }
+        },
+        "required": [
+          "source_format",
+          "size_kilobytes"
+        ],
+        "additionalProperties": false
```

Comment:
Could the source info include a checksum?

A URL to the source comes to mind but I think we are going to do something like https://arxiv.org/abs/1234.12345/src?

Reply:
added